### DISCUSSION TITLE
Release Cadence

### DISCUSSION THREAD

**Discussion by j-bahr** [microsoft/gctoolkit#232] [Q&A] [answered]
I wanted to post a quick question for @kirk-microsoft or @karianna to see what the release cadence we should expect for gctoolkit. I'm happy to contribute improvements and am hoping once upstreamed I can use the improvements by pulling versioned artifacts from maven central. Is this just an ad-hoc process now?

**Comment by ghost** [comment, marked as answer]
The release cadence is adhoc and will likely remain so. At the moment, I'm at the beginning of a refactoring that should break a circular dependency between the vertx and api modules. Translation, I'm unlikely to push a new version until the refactoring has been completed. That said, I'm very happy to push groups of useful changes up to maven central upon request.

**Reply by j-bahr** [reply]
Thanks for the info, I'll keep this in mind going forward. I can keep local changes for a while but will eventually need to pick them up through a versioned release. 

Thanks!

**Reply by j-bahr** [reply]
@kirk-microsoft, just following back up here. It's been about 2 months since I last asked about this. Are you any closer on landing the refactor?  If not would it be possible to get a maintenance release pushed to maven central so that folks like me can pick up the ZGC changes I contributed back in Sept?

**Comment by ghost** [comment]
Hi Jeff,

I’ll push.

Kind regards,
Kirk


From: Jeff Bahr ***@***.***>
Date: Tuesday, November 1, 2022 at 3:02 PM
To: microsoft/gctoolkit ***@***.***>
Cc: Kirk Pepperdine ***@***.***>, Mention ***@***.***>
Subject: Re: [microsoft/gctoolkit] Release Cadence (Discussion #232)

@kirk-microsoft<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fkirk-microsoft&data=05%7C01%7CKirk.Pepperdine%40microsoft.com%7C47a3a9e0071b4190e6be08dabc54cbc7%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638029369627952578%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=2XGkXtbKfl3nX2e107lIHfs86LkE%2BDyI6abjxIWey%2Fg%3D&reserved=0>, just following back up here. It's been about 2 months since I last asked about this. Are you any closer on landing the refactor? If not would it be possible to get a maintenance release pushed to maven central so that folks like me can pick up the ZGC changes I contributed back in Sept?

**Reply by karianna** [reply]
2.0.11 has been pushed :-)
